odinmain/src/main/kotlin/me/odinmain/features/impl/dungeon/BloodCamp.kt

```diff
@@ -177,15 +178,15 @@ object BloodCamp : Module(
 
         val ping = ServerUtils.averagePing
 
-        forRender.filter { (entity) -> !entity.isDead }.forEach { (entity, data) ->
+        forRender.filter { (entity) -> !entity.isDead }.forEach { (entity, renderData) ->
```

Comment:
just use it in the filter

Reply:
not entirely sure what you mean

Reply:
forRender.filter { !it.isDead }.blahblah

Reply:
it doesnt work here. I think its because i use a map.